### DISCUSSION TITLE
手机端的播放界面有溢出。

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [AlistGo/alist#4869] [Invalid question]
![IMG_6269](https://github.com/alist-org/alist/assets/78125220/6fa67c50-0070-4653-ac16-31926d5e8858)
手机端的视频播放界面有溢出，希望能尽快解决，都好几个版本有这个问题了，还有选择播放器和下载按钮希望能汉化，音乐播放器希望能换一个好看的ui

**Comment by Dashan-IZ** [comment]
网站：http://ucloud.cool/

**Comment by Dashan-IZ** [comment]
还有问一下有没有能够加速国外服务器的，对中国延迟很好的免费cdn，希望知道的能给我个名字。谢谢

**Reply by JJTools** [reply]
不可能有CDN 给你白嫖的价钱吧。国内流量贵。垃圾三流CDN你也看不上。

**Comment by anwen-anyi** [comment]
> ```css
> /*artplayer video*/
> .art-video-player .art-bottom .art-controls .art-control {
>   min-width: 0px !important;
> }
> ```
> 
> disappointed_relieved 不知道这样对不对，测试没发现有啥影响（其他小伙伴有需要的自取吧）

因为open播放器多一个画质选择，暂时只能添加这个css解决一下

**Reply by Dashan-IZ** [reply]
请问这段css代码添加在那个地方？

**Reply by Dashan-IZ** [reply]
请问这段css代码添加在那个地方？
